src/aggregate/aggregate_request.c

```diff
@@ -986,10 +992,13 @@ int AREQ_ApplyContext(AREQ *req, RedisSearchCtx *sctx, QueryError *status) {
     }
   }
 
-  if (opts->language == RS_LANG_UNSUPPORTED) {
+  if (opts->language == RS_LANG_INDEX_DEFAULT) {
+    opts->language = index->rule->lang_default;
```

Comment:
What is the default initialized to?
If it's not set by the user, is it English?

Reply:
Yes, if it is not set by the user, the default language is English